server/index.ts

```diff
@@ -185,7 +185,9 @@ app.get(`${base}/status`, (_, res) => {
 });
 
 app.get(`${base}/web-manifest`, (req, res) => {
-  const startUrl = `${req.protocol}://${req.get('host')}${base}`;
+  const host = req.get('host');
+  const isLocalhost = host?.startsWith('localhost');
+  const startUrl = `${isLocalhost ? req.protocol : 'https'}://${host}${base}`;
```

Comment:
Only a suggestion - should we use here `process.env.NODE_ENV` to check it's a development environment then use `http`?

Reply:
good point 👍